Cargo.toml

```diff
@@ -44,7 +44,6 @@ bigdecimal = "0.2.0"
 smallvec = "1.6.1"
 atty = "0.2.14"
 url = "2.2.2"
-reqwest = "0.11.3"
```

Comment:
This is a high level library with a bunch of dependencies. It's not worth it for one command, and I wasn't actually thrilled with the idea of that command in the first place.